### ISSUE TITLE
DataPath on Linux (Help Q_Q)

### ISSUE THREAD

**Issue by WoweePaw** [SapphireServer/Sapphire#695]
**Describe the problem:**
I'm unable to get the path "right"... for some reason it don't accepts "./sqpack" (folder copied into the root of the Servers directory)

Am I'm doing it wrong? The Linux Wiki is not very detailed about this part... :/

**Screenshots or videos needed to identify the problem:**
Nothing to show... just the "DataPath wrong" error in console

**Sapphire branch and commit:**
heads/develop with commit b357e40f

aka

"latest" (so far) ;)

**Logs:**
[world_2020-08-24.log](https://github.com/SapphireServer/Sapphire/files/5117412/world_2020-08-24.log)

**Setup:**
- Ubuntu 18.04 (everything up-to date)
- g++8
- gcc8
- cmake (latest)
- Final Fantasy (updated yesterday)

Hope someone can help... just wanted to try this out QwQ

**Comment by NotAdam** [comment]
If you're pointing it at a 5.3 client it won't work, sheets need to be updated

**Comment by D3thw0lf** [comment]
Oh. Is there a way to "backpatch" the Client to 5.2 or older or do you recommend just to wait until it's ready? I'm very new to the Final Fantasy Client/Server thing ^^'

**Comment by NotAdam** [comment]
Not really, you'd need to patch a new client from scratch to get back to 5.28. There's some discussion regarding this in the discord which might be of some use, but there's not exactly an easy way of doing this, not yet anyway.

**Comment by D3thw0lf** [comment]
Alright then. Thanks for the fast responses and I'll try that then ^^.

Cya and have a nice day/night. ❤️

**Comment by collett8192** [comment]
This as well as #693 can be closed since we are once again on the latest version (until next month).

**Comment by silvernode** [comment]
Having this error as well and the path is correct. My question is, will the error be vague like OP reported when the current game patch isn't supported yet or would I get a specific error stating that I need to wait for support? My guess is that it simply won't be able to load the data from an unsupported sqpack and report an error as such until the server is brought up to speed with the current version of the game.

**Comment by Kooper16** [comment]
I know this is old but I had the same issue. It wants the sqpack path but it actually also needs the files that are on the same level as the sqpack folder (movie folder, ffxivgame.ver, etc.). I don't know if it needs all those files but at least some of them. So the fix is to copy the whole "game" folder (the one next to the boot folder).
